server/tournaments/generator-elimination.ts

```diff
@@ -66,21 +73,19 @@ class ElimNode {
 		}
 		this.children = children;
 	}
-	traverse(/** @type {(node: ElimNode) => void} */ callback) {
-		/** @type {ElimNode[]} */
-		const queue = [this];
+	traverse(callback: (node: ElimNode) => void) {
+		const queue: ElimNode[] = [this];
 		let node;
+		// tslint:disable-next-line: no-conditional-assignment
```

Comment:
Man, tslint doesn't have the double parens to get around conditional assignment? Rude.